libra-node/src/lib.rs

```diff
@@ -220,6 +220,46 @@ fn setup_debug_interface(config: &NodeConfig, logger: Option<Arc<Logger>>) -> No
     NodeDebugService::new(addr, logger)
 }
 
+async fn periodic_state_dumb(node_config: NodeConfig, db: DbReaderWriter) {
+    use futures::stream::StreamExt;
+
+    let args: Vec<String> = ::std::env::args().collect();
+
+    // Once an hour
+    let mut config_interval = tokio::time::interval(std::time::Duration::from_secs(60 * 60)).fuse();
```

Comment:
I wonder does the compiler precompile the 60 * 60 in this state, and store 3600?

Reply:
yes it would